docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -334,61 +371,32 @@ children:
         url: Appsody-LoopBack.html
         output: 'web, pdf'
 
-  - title: 'Using Express Middleware'
-    url: Express-middleware.html
-    output: 'web, pdf'
-
-  - title: 'Serving Static Files'
-    url: Serving-static-files.html
-    output: 'web, pdf'
-
-  - title: 'Dynamically adding models, repositories, and controllers'
-    url: Dynamic-models-repositories-controllers.html
-    output: 'web, pdf'
-
-  - title: 'Uploading and Downloading Files'
-    url: File-upload-download.html
-    output: 'web, pdf'
-
-  - title: 'Validation'
-    url: Validation.html
+  - title: 'Running and Debugging Apps'
     output: 'web, pdf'
     children:
-      - title: 'Validation in REST Layer'
-        url: Validation-REST-layer.html
-        output: 'web, pdf'
-
-      - title: 'Validation in the Controller, Repository and Service Layer'
-        url: Validation-controller-repo-service-layer.html
-        output: 'web, pdf'
-
-      - title: 'Validation in ORM Layer'
-        url: Validation-ORM-layer.html
-        output: 'web, pdf'
-
-  - title: 'Self-hosted REST API Explorer'
-    url: Self-hosted-rest-api-explorer.html
-    output: 'web, pdf'
 
-  - title: 'Creating CRUD REST APIs from a model'
-    url: Creating-crud-rest-apis.html
-    output: 'web, pdf'
+    - title: 'Setting debug strings'
+      url: Setting-debug-strings.html
+      output: 'web, pdf'
 
-  - title: 'Boot and Mount a LoopBack 3 Application'
-    url: Boot-and-Mount-a-LoopBack-3-application.html
-    output: 'web, pdf'
+    - title: 'Using strong-error-handler'
+      url: Using-strong-error-handler.html
```

Comment:
I am proposing to move strong-error-handler docs to "Building REST APIs", because it's applicable to REST APIs only (as opposed to e.g. building pubsub/messaging style projects communicating via Kafka).